### DISCUSSION TITLE
convert multiple pngs (sit in different folder path) to jpeg

### DISCUSSION THREAD

**Discussion by unicorneh** [ImageMagick/ImageMagick#2840] [Help] [answered]
First time using this wonderful script, I can convert single png to jpeg, no problem here.

But now, I have multiple pngs, in location like this:
\\dpc\proj\shield\design\rough\v001.png
\\dpc\proj\glass\design\colour\v003a.png
\\dpc\proj\wheel\design\clean\v023.png
\\dpc\proj\balls\design\build\v076a.png
\\dpc\proj\macy\design\colour\v0991.png

and I need to convert each of them to this location:
\\dpc\proj\shield\design\rough\JPEG
\\dpc\proj\glass\design\colour\JPEG
\\dpc\proj\wheel\design\clean\JPEG
\\dpc\proj\balls\design\build\JPEG
\\dpc\proj\macy\design\colour\JPEG

is there a way to do so with Windows PowerShell with one single command?

Thank you,

**Comment by snibgo** [comment]
I think your question is about Powershell, and how to loop through folders. If so, I suggest you consult Powershell documentation.

**Reply by DocDJ** [reply]
I have a similar problem, but a bit more difficult. Given a SOURCE folder "E:\xx\image-in" with hundreds of jpg files and one OUTPUT "E:\xx\image-out" folder, I have tried to use Magick's globbing ability to apply the  same action to each of those images.  I do NOT want to use batch files to do it, because that means continuously restarting Magick for EACH image. Every command string I try, I get an error message saying "-path" is an unrecognized option "ProcessCommandOptions/428".  I do not want to merge the images. One file out for each file in. 
PS. I am trying to apply the Caption metadata for each image to its output  image.

**Reply by snibgo** [reply]
You have posted to a thread that has been answered, ie closed.

I suggest you start a new thread for your question. Please state what version of IM, on what platform, and what command(s) you have used.

**Comment by unicorneh** [comment, marked as answer]
answered my own question:
add "; " after each magick
